File: project/src/main/java/com/example/demo/infrastructure/security/configuration/authentication/RegisterRequest.java
package com.example.demo.infrastructure.security.configuration.authentication;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {

    // Representación del DTO
    private String nombres;
    private String apellidos;
    private Integer tipoUsuarioId;
    private String apodo;
    private String direccion;
    private Integer edad;
    private String celular;
    private String dni;
    private String email;
    private String password;
}

File: UYWALKYP/app/src/main/java/com/delvin/uywalkyp/RegisterSchema/RegisterRequest.kt
package com.delvin.uywalkyp.RegisterSchema

data class RegisterRequest(
    val apellidos: String,
    val apodo: String,
    val celular: String,
    val direccion: String,
    val dni: String,
    val edad: Int,
    val email: String,
    val nombres: String,
    val password: String,
    val tipoUsuarioId: Int
)